diff --git a/user/admin.py b/user/admin.py
index b5e9d88..303c46c 100644
--- a/user/admin.py
+++ b/user/admin.py
@@ -2,14 +2,14 @@ from django.contrib import admin
 from django.contrib.auth import admin as auth_admin
 
 from .forms import UserChangeForm, UserCreationForm
-from .models import User
-# o admin esta exatamente igual so precisa ser personalizado
+from .models import User, Employee
+
+class EmployeeInline(admin.StackedInline):
+    model = Employee
+    can_delete = True
+    verbose_name_plural = 'employee'
 
 @admin.register(User)
 class UserAdmin(auth_admin.UserAdmin):
-    form = UserChangeForm
-    add_form = UserCreationForm
-    model = User
-    fieldsets = auth_admin.UserAdmin.fieldsets + (
-        ("Informações Pessoais", {"fields": ("bio",)}),
-    )
+    inlines = (EmployeeInline,)
+
