diff --git a/user/management/commands/create_groups.py b/user/management/commands/create_groups.py
new file mode 100644
index 0000000..e5444ab
--- /dev/null
+++ b/user/management/commands/create_groups.py
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group, Permission
+
+class Command(BaseCommand):
+    help = 'Create base groups'
+
+    def handle(self, *args, **kwargs):
+        new_group, created = Group.objects.get_or_create(name='Nutritionist')
+        proj_add_perm = Permission.objects.get(codename='view_clinicevaluation')
+        new_group.permissions.add(proj_add_perm)
